tst/working/quick/ClassicalNaturalNaming.tst

```diff
@@ -29,12 +29,12 @@ gap> d:=7;; for q in [2, 4, 5, 7, 8, 9, 11, 13] do TestNaming("SO", d, q); od;
 #
 # O^+(4,q) with q = 8 or q >= 11 are non-generic
 #
-# FIXME/TODO: sometimes get SO(1,4,11) has bad value for isSOContained; expected true, got unknown
-# FIXME/TODO: sometimes get SO(1,4,13) has bad value for isSOContained; expected true, got unknown
-# FIXME/TODO: sometimes get SO(1,8,4) has bad value for isSOContained; expected true, got unknown
+# FIXME/TODO: sometimes get "SO(1,d,q) has bad value for isSOContained;
+# expected true, got unknown" for
+# (d,q) in [ (4,11), (4,13), (8,4), (8,7), (8,8) ]
```

Comment:
I took the liberty of cherry picking the commit "Remove broken tests in ClassicalNaturalNaming.tst" to master, as it affects other PRs.